README.md

```diff
@@ -0,0 +1,53 @@
+<a name="readme-top"></a>
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+- [Setup](#setup)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+
+
+# 📖 [Hello-Microverse] <a name="about-project"></a>
+
+> First project for microverse Module -1, week -1 project exercise
+
+
+### Key Features <a name="key-features"></a>
+
+- **[gitignore file added]**
+- **[html-css linter]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Setup
+Clone this repository to your desired folder:
+
+git clone "https://github.com/ram1117/hello-microverse.git"
+
+## 👥 Authors <a name="authors"></a>
+
+> Ram Kumar Karuppusamy
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/ram1117)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
```

Comment:
Nice work using the lates readme template. :+1:  However, kindly make sure that all sections/headers of the readme are present and update according to your project and credentials. :pray: